Buttons.d

```diff
@@ -326,6 +305,29 @@ func int Button_GetCaptionPtr(var int hndl) {
 	return v.textLines_next;
 };
 
+func void _Button_processMouseEvents( var int btn_hndl ) {
+	var _Button btn; btn = get( btn_hndl );
+
+	var int CY; CY = Print_ToVirtual(CURSOR_Y, PS_Y);
+	var int CX; CX = Print_ToVirtual(CURSOR_X, PS_X);
+	if (btn.state & BUTTON_ACTIVE) {
+		if (btn.posx <= CX && btn.posx2 >= CX && btn.posy <= CY && btn.posy2 >= CY) {
+			if (Cursor_Left==KEY_PRESSED) {
+				MEM_PushIntParam(btn_hndl);
+				MEM_CallByID(btn.on_click);
+			};
+			if ((btn.state & BUTTON_ENTERED)==0) {
+				MEM_PushIntParam(btn_hndl);
+				MEM_CallByID(btn.on_enter);
+				btn.state = btn.state | BUTTON_ENTERED;
+			};
+		} else if (btn.state & BUTTON_ENTERED) {
+			MEM_PushIntParam(btn_hndl);
+			MEM_CallByID(btn.on_leave);
+			btn.state = btn.state & ~BUTTON_ENTERED;
+		}; 
+	};
+};
```

Comment:
Add return for `foreachHndl`

```D
    return rContinue;
};
```

Reply:
what is rContinue?

Reply:
It's essentially `continue` with a different name (to not confuse them with the Ikarus jump symbols I assume):

https://github.com/Lehona/LeGo/blob/d76cbccf211a2816076a80d4bc7fd2d72253d694/PermMem.d#L183-L188